docs/why-dbos.md

```diff
@@ -1,38 +1,149 @@
 ---
-hide_table_of_contents: true
+hide_table_of_contents: false
+hide_title: false
+title: Why DBOS?
 ---
 
-# Why DBOS?
+<p style={{ fontSize: '20px', color: 'var(--ifm-color-primary-darkest)' }}>DBOS is an open-source library for building **reliable** and **fault-tolerant** applications.</p>
 
-DBOS helps you write code that is **reliable** and **fault-tolerant** by default.
-DBOS **durably executes** your programs, so that if they fail, upon restart they automatically resume from where they left off.
-This makes all sorts of programs easier to write:
+Let's look at a common situation where reliability is critical.
+Imagine you're running an e-commerce platform where an order goes through multiple steps:
 
+<img src={require('@site/static/img/why-dbos/workflow-example.png').default} alt="Durable Workflow" width="750" className="custom-img"/>
 
-<Tabs groupId="examples">
+This program looks simple, but making it _reliable_ is deceptively difficult.
+Here are some potential problems:
+
+- Your program crashes after step 1, "Validate Payment". The customer has been charged, but their order is never shipped.
+- You get to step 2, "Check Inventory", and you're out of stock. You need to wait 24 hours for the new inventory before you can ship your order. You need that step to sleep for a day.
+
+DBOS makes those problems easier to solve because you can add decorators like `DBOS.workflow()` and `DBOS.step()` to your program:
```

Comment:
```suggestion
These situations can cause hours (or even days!) of debugging and can require deep distributed systems expertise to solve.

DBOS makes those problems easier to solve because you can add decorators like `DBOS.workflow()` and `DBOS.step()` to your program:
```